### ISSUE TITLE
Switch/case indentation for Swift files in xCode 13.1

### ISSUE THREAD

**Issue by damien-nd** [realm/SwiftLint#3759]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

I set xCode 13.1 to indent switch case for Swift file, and the behavior is now a tabulation before each case.
=> Is it a new "best practice" and we should update SwiftLint?

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
```

warning: Switch and Case Statement Alignment Violation: Case statements should vertically align with their enclosing switch statement. (switch_case_alignment)

### Environment

* SwiftLint version: 0.45.0
* Installation method used: Homebrew
* No configuration file (default options)
* Which Xcode version are you using: 13.1
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
switch true {
    case true : print("true") // This triggers a violation
    case false : print("false") // This triggers a violation
}
```

**Comment by jpsim** [comment]
The rule is working as expected here.
